documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+It is thus recommended to execute all DDLs in a single batch so the procedure is done only once.
+====
+
+[NOTE]
+====
+Not all schema changes are supported when CDC is enabled for a source table.
+We note here some of the likely effects:
+
+- If a column name is changed then the old column will continue to be used by the ASN capture service and therefore the new name will not appear in Debezium.
+If Debezium is restarted then the new name will appear.
+- To Be Completed
+
+It is recommend that if the structure of a source change is changed, that we:
+- that we mark the tables as inactive on the ASN register table
+- reinit the ASN capture service (see the UDFs)
+- update the ASN representation of the table (manual task)
+- mark the table as active
+- reinit the ASN capture service again (see the UDFs)
+====
+
+==== Cold schema update
+
+This is the safest procedure but might not be feasible for applications with high-availability requirements.
+The administrator should follow this sequence of steps:
+
+1. Suspend the application that generates the database records
+2. Wait for Debezium to stream all unstreamed changes
+3. Stop Debezium connector
+4. Apply all changes to the source table schema
+5. Mark the tables as INACTIVE on the ASN register table and reinit the ASN capture service (see the UDFs)
+6. Remove the old structure table from ASN
+7. Add the new structure table to ASN
+8. Mark the tables as ACTIVE on the ASN register table and reinit the ASN capture service (see the UDFs)
+9. Resume the application
+10. Start Debezium connector
+
+
+==== Hot schema update
+
+The hot schema update does not require any downtime in application and data processing.
+The procedure itself is also much simpler than in case of cold schema update.
+First we consider an incremental change to the source, e.g. adding a new column to the end:
+
+1. lock the source table to change
+2.  Mark the tables as INACTIVE on the ASN register table and reinit the ASN capture service (see the UDFs)34. Apply all changes to the source table schema
+3.  Apply all changes to the ASN table schema
+4.  Mark the tables as ACTIVE on the ASN register table and reinit the ASN capture service (see the UDFs)
+
+
+Now we consider an non-incremental change to the source, e.g. adding a new column in the middle:
+
+1. lock the source table to change
+2.  Mark the tables as INACTIVE on the ASN register table and reinit the ASN capture service (see the UDFs)
+3. export the data of the source table to change
+4. truncate the source table
+5. alter the source table
+6. LOAD the exported data into the altered source table 
+7. export the data of the ASN table to change
+8. truncate the ASN table
+9. alter the ASN table
+10. LOAD the exported data into the altered ASN table 
+11. Mark the tables as INACTIVE on the ASN register table and reinit the ASN capture service (see the UDFs)
+
+
+==== Example
+
+To Be Done
```

Comment:
Are you planning to do this as part of this PR?

Reply:
No I don't think we will have time,

Reply:
Ok, I've added this to the list of follow-up tasks in [DBZ-1752](https://issues.redhat.com/browse/DBZ-1752).